src/lib/components/column-profile/CollapsibleTableHeader.svelte

```diff
@@ -124,7 +105,7 @@
           <span
             class="grid grid-flow-col gap-x-2 text-gray-500 text-clip overflow-hidden whitespace-nowrap "
           >
-            {#if titleElementHovered || emphasizeTitle}
+            {#if titleElementHovered || emphasizeTitle || contextMenuOpen}
```

Comment:
see similar comment below about pulling business logic out of rendering code -- in a case like this, once we get up to 3 bools, we now have 8 possible boolean states that we'd need to check on if we wrote a test around this rendering code. That much rendering logic is a lot to test, b/c testing rendering sucks -- needs all kinds of harnesses and mocks and stuff.

On the other hand, we could roll these three booleans into a single boolean  `showEntityDetails = titleElementHovered || emphasizeTitle || contextMenuOpen`  in the `<script>` section of this component, in which case there are just two UI states to test, and the 8 states of the boolean can be tested in plain code.

This is kind of a silly example, because you probably wouldn't test the states of and 8 state boolean that is defined like this, since the test would basically match the defintion. And this might seem a bit pedantic, because... yes, I know: we have almost no UI component tests today. So I'm not actually asking you to change this code in this particular instance :-) I just noticed it as a pattern that I see a lot in the codebase (a lot of logic in presentational code), and I'm starting to look down the road to the day when hopefully we _do start testing the UI_, so I thought I'd point out this pattern and suggest an option that a lot of people use to simplify UI testing. It's just a thought, for whatever that's worth, :-)